scripts/armadillo-setup.sh

```diff
@@ -0,0 +1,298 @@
+  
+  echo "Config downloaded"
+
+}
+setup_nginx_config() {
+  wget -q -O /etc/nginx/sites-enabled/armadillo.conf https://raw.githubusercontent.com/molgenis/molgenis-service-armadillo/scripts/conf/armadillo-nginx.conf
+  
+  sed -i -e 's/@ARMADILLO_URL@/'"$ARMADILLO_DOMAIN"'/' /etc/nginx/sites-enabled/armadillo.conf
+  systemctl reload nginx
+  echo "Config downloaded"
+
+}
+
+download_armadillo() {
+  
+  if [ -z "$ARMADILLO_VERSION" ]; then
+    LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' -s $ARMADILLO_URL/releases/latest)
+    #ARMADILLO_VERSION=$(echo "$LATEST_RELEASE" | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+    ARMADILLO_VERSION=$(echo "$LATEST_RELEASE" | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+    
+    if [[ "$ARMADILLO_VERSION" =~ 'armadillo-service-2' ]]; then
+      echo "Armadillo version 2 not supported! Please use provide an armadillo 3 version with --version"
+      exit 1;
+    fi
+    DL_URL=https://github.com/molgenis/molgenis-service-armadillo/releases/download/armadillo-service-$ARMADILLO_VERSION/armadillo-$ARMADILLO_VERSION.jar
+  else
+    DL_URL=https://github.com/molgenis/molgenis-service-armadillo/releases/download/armadillo-service-$ARMADILLO_VERSION/armadillo-$ARMADILLO_VERSION.jar
+    
+  fi 
+  wget -q -O $ARMADILLO_PATH/application/armadillo-"$ARMADILLO_VERSION".jar "$DL_URL"
+  ln -s $ARMADILLO_PATH/application/armadillo-"$ARMADILLO_VERSION".jar $ARMADILLO_PATH/application/armadillo.jar
+  echo "$ARMADILLO_VERSION downloaded"
+}
+
+
+check_req() {
+  for COMMAND in "nginx" "java" "wget" "docker" "curl" "whoami"; do
+    command_exists "${COMMAND}"
+  done
+
+  if [ "$(whoami)" != 'root' ]; then
+    echo 'Please run this script with root or sudo rights!'
+    exit 1;
+  fi
+
+}
+
+
+
+command_exists() {
+    # check if command exists and fail otherwise
+    command -v "$1" >/dev/null 2>&1
+    if [ $? -ne 0 ]; then
+        echo "I require $1 but it's not installed. Abort."
+        exit 1
+    fi
+}
+
+
+cleanup(){
+  if [ "$ARMADILLO_CLEANUP" ]; then
+    echo "--cleanup variable defined! Are you sure. Potential data loss? Type y/N"
+    read -r -p "Type y/N" response
+      if [ "$response" = "y" ]
+        then
+            systemctl stop armadillo
+            systemctl disable armadillo
+            rm -Rf $ARMADILLO_PATH
+            rm -Rf $ARMADILLO_LOG_PATH
+            userdel $ARMADILLO_SYS_USER
+            rm -Rf /etc/systemd/system/armadillo.service
+            systemctl daemon-reload
+            rm -Rf $ARMADILLO_LOG_PATH
+            rm /etc/nginx/sites-enabled/armadillo.conf
+            systemctl reload nginx
+            echo "Armadillo cleaned!"
+        else
+          echo "No cleanup .. please remove the --cleanup argument"
+          exit 0
+      fi
+  fi
+}
+
+startup_armadillo() {
+  systemctl enable armadillo
+  systemctl start armadillo
+  echo "Armadillo started"
+
+}
+
+#Parameters passed in help
+parameters_help() {
```

Comment:
The `--cleanup` argument is missing in the help

Reply:
The idea was that this shoud be an hidden feature.. (Describe it in the documentation as last resort of uninstall part, or use it as an DevOps process)